dropletutils-read-10x-counts.R

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env Rscript 
```

Comment:
Remember you don't need to duplicate this script. Use it from the dropletutils-scripts package :-)